### PR TITLE
Update param docs

### PR DESCRIPTION
**Pull request by ErinCain**

### CODE REVIEW

File: data-raw/cache-data.R [hunk 1]
```diff
@@ -18,44 +19,42 @@ adult_seeds[ , 1] <- adult_seed_values
 rownames(adult_seeds) <- DSMhabitat::watershed_species_present$watershed_name[-32]
 usethis::use_data(adult_seeds, overwrite = TRUE)
 
+# Watershed Labels 
+watershed_labels <- c("Upper Sacramento River", "Antelope Creek", "Battle Creek",
```
Comment (thesadie) [inline data-raw/cache-data.R]: if you are going to add this, I think you should use it on to set `rownames(adult_seeds)` above. so maybe move this to the top of the script

File: data-raw/model-parameters.R [hunk 2]
```diff
@@ -160,18 +160,14 @@ params <- list(
...
-  ..surv_juv_rear_prop_diversions = -0.1755,
-  ..surv_juv_rear_total_diversions = -0.0005,
+  ..surv_juv_rear_contact_points = 0.0358,
+  ..surv_juv_rear_prop_diversions = 0.05,
+  ..surv_juv_rear_total_diversions = 0.215,
   ..surv_juv_bypass_int = -3.5,
   ..surv_juv_delta_int = 1.4,
-  ..surv_juv_delta_contact_points = (.0358 * -0.189),
-  ..surv_juv_delta_total_diverted = (.5 * -0.0021),
+  ..surv_juv_delta_contact_points = .0358,
```
Comment (thesadie) [inline data-raw/model-parameters.R]: where is the -0.189 part now?

Comment (thesadie) [reply data-raw/model-parameters.R]: and for the others that were products before

File: data-raw/cache-data.R (hunk 1 above)
Comment (ErinCain) [reply data-raw/cache-data.R]: moved!

File: data-raw/model-parameters.R (hunk 2 above)
Comment (ErinCain) [reply data-raw/model-parameters.R]: moved up into submodule. Will reorganize to move all calibrated parameters to submodules.

**Comment by ErinCain** [comment]
I fixed the pkgdown links. It ended up being a different problem than I originally thought.  I had to change the link in the `DSMhabitat` `DESCRIPTION` so that it pointed to pkgdown site not github url. Rebuilding `DSMhabitat` pkgdown site this way and then rebuilding late fall run site fixes the links.

**Pull request merged**